### PR TITLE
Automated testing

### PR DESCRIPTION
**Pull request by Khegyi**

### CODE REVIEW

File: test/AdvancedGridList.test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    openFunction: jest.fn(),
+    imgList: [],
+  }
+  it('Matches snapshot', () => {
+    const wrapper = shallow(<AdvancedGridList {...testprops} />)
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  describe('pickTile', () => {
+    it('should return 1 or 2', () => {
```
Comment (B3zo0) [inline test/AdvancedGridList.test.tsx]: should return 1 or 2 but this test only checks for 2.

File: test/DotsMobileStepper.test.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { mount } from 'enzyme'
+import React from 'react'
+import { Button } from '@material-ui/core'
+import { DotsMobileStepper } from '../src/components/DotsMobileStepper'
+
+describe('handleNext', () => {
+  const proba = {
```
Comment (B3zo0) [inline test/DotsMobileStepper.test.tsx]:
```suggestion
  const mock = {
```
can you please use English variable names?

File: test/AdvancedGridList.test.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import React from 'react'
+import { AdvancedGridList, pickTile } from '../src/components/AdvancedGridList'
+
+describe('AdvancedGridList', () => {
+  const testprops = {
+    openFunction: jest.fn(),
+    imgList: [],
+  }
+  it('Matches snapshot', () => {
+    const wrapper = shallow(<AdvancedGridList {...testprops} />)
```
Comment (B3zo0) [inline test/AdvancedGridList.test.tsx]: The snapshot is not showing anything. Add 2 images to the imgList.

File: yarn.lock [hunk 4]
```diff
@@ -105,6 +105,13 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
+"@babel/runtime@^7.4.4":
```
Comment (B3zo0) [inline yarn.lock]: Please remove this file.

File: test/DotsMobileStepper.test.tsx (hunk 2 above)
Comment (Khegyi) [reply test/DotsMobileStepper.test.tsx]: Sure thing

File: yarn.lock (hunk 4 above)
Comment (Khegyi) [reply yarn.lock]: done

File: package.json [hunk 5]
```diff
@@ -73,8 +73,9 @@
     "@sensenet/client-core": "^2.1.0",
     "@sensenet/default-content-types": "^1.2.2",
     "moment": "^2.24.0",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
+    "yarn": "^1.17.3"
```
[the remark this answers is not in the document]
Comment (Khegyi) [reply package.json]: Removed

File: test/AdvancedGridList.test.tsx (hunk 1 above)
Comment (Khegyi) [reply test/AdvancedGridList.test.tsx]: fixed

File: test/AdvancedGridList.test.tsx (hunk 3 above)
Comment (Khegyi) [reply test/AdvancedGridList.test.tsx]: Done

**Pull request merged**